grails-app/domain/org/pih/warehouse/order/OrderAdjustment.groovy

```diff
@@ -155,16 +174,13 @@ class OrderAdjustment implements Serializable, Comparable<OrderAdjustment> {
      *  - if adjustment has no regular invoice yet and order is placed
      * */
     Boolean isInvoiceable() {
-        if (canceled) {
-            return hasPrepaymentInvoice && !hasRegularInvoice && order.placed
-        }
+        Boolean invoicedFullAmount = invoicedAmount == totalAdjustments
 
-        // If is not canceled and has regular invoice, check if by any chance it was invoiced as cancelled
-        if (!canceled && hasRegularInvoice) {
-            return invoicedQuantity == 0 && order.placed
+        if (canceled) {
+            return hasPrepaymentInvoice && !hasRegularInvoice && order.placed && !invoicedFullAmount
         }
 
-        return !hasRegularInvoice && order.placed
+        return order.placed && !invoicedFullAmount
```

Comment:
Don't we need to change the comment above about when the order adjustment is invoiceable?